jupyter/datascience/ubi9-python-3.9/utils/processor_kfp.patch

```diff
@@ -8,7 +8,25 @@
                  )
              else:
                  client = ArgoClient(
-@@ -416,7 +417,7 @@
+@@ -309,8 +310,16 @@
+ 
+                 # Create an instance id that will be used to store
+                 # the pipelines' dependencies, if applicable
+-                pipeline_instance_id = f"{pipeline_name}-{timestamp}"
++                pipeline_instance_id = ""
+ 
++                # Use environment variable to set version as suffix instead of timestamp
++                if os.environ.get("KFP_SUFFIX_USE_VERSION") == "true" :
++                    # Version is determined by the count of existing pipeline versions
++                    version = client.list_pipeline_versions(pipeline_id=pipeline_id).total_size
```

Comment:
We need to check this pattern,
the scenario, which i would like to test is following:
- once we have few pipeline. ex: 10 , the version=10
- a user delete the 6th pipeline, the total_size=9 
- the next call of pipeline, would try to create the version=10 
again , perhaps. WDYT?

Reply:
I hadn't thought of deletion case, the current patch will fail to handle a deletion scenario. Let me search the docs a bit more and try to find a better approach.